### ISSUE TITLE
ACK messages are malformed

### ISSUE THREAD

**Issue by akoutmos** [HCA-Healthcare/elixir-mllp#4]
Firstly, hello and thank you for the awesome library!

I am currently using this library and noticed that ACK HL7 messages and not in line with the HL7 spec. Specifically, the ACK payload is not prefixed with `<0x0B>` and postfixed with `<0x1C><0x0D>`. This is also evident in the Wireshark capture as Wireshark does not recognize the ACK as an HL7 message (screen shots are dummy data not PHI lol):

![image](https://user-images.githubusercontent.com/4753634/90775725-c7aeb280-e2be-11ea-91cf-9f1dca0513fd.png)

Compared to a properly formatted HL7 ACK message:

![image](https://user-images.githubusercontent.com/4753634/90775767-d7c69200-e2be-11ea-8ae9-b64db7365ccb.png)

I forked the library and made the necessary adjustments (in a quick and dirty fashion) https://github.com/akoutmos/elixir-mllp/commit/3ae4ab26ef48db87f61127f0e16b3ba3389961d0

Thoughts?

**Comment by bryanhunter** [comment]
Initially this was a head-scratcher because it is working correctly for us in our projects. After digging in, we realized it was a problem with the readme docs and with the throw-away DefaultDispatcher's implementation. In both cases, we were missing the needed call to `MLLP.Envelope.wrap_message` on the reply. The dispatchers we use in our production projects have that line. 

Thanks for spotting the gap @akoutmos!

Fixed in #6
